Mixpanel/Mixpanel.m

```diff
@@ -423,6 +425,26 @@ - (void)track:(NSString *)event properties:(NSDictionary *)properties
 #endif
 }
 
+#if !MIXPANEL_NO_NOTIFICATION_AB_TEST_SUPPORT
+- (void)trackPushNotificationAutomatically {
```

Comment:
can you rename this to something that makes it clear that it sets up automatic tracking? before reading it i thought it was what you would call to track once